web_ui/src/pages/project-details/components/project-models/models-container/model-card/model-info-fields.component.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2022-2025 Intel Corporation
+// LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+import { Text } from '@geti/ui';
+
+import { ThreeDotsFlashing } from '../../../../../../shared/components/three-dots-flashing/three-dots-flashing.component';
+
+import classes from './model-card.module.scss';
+
+export const ModelInfoFields = ({
+    modelSize,
+    totalDiskSize,
+    complexity,
+    isModelTraining,
+}: {
+    modelSize?: string | undefined;
```

Comment:
Since the prop is already marked with `?`, it implicitly allows `undefined`, so there's no need to include `undefined` in the type explicitly